static/js/jira-select-github-version.js

```diff
@@ -0,0 +1,17 @@
+$('.jiraSelectGitHubVersion__options__card').click(function (event) {
+	event.preventDefault();
+	const selectedCard = event.target.className;
+	const selectedVersion = selectedCard.substring(selectedCard.indexOf(' ') + 1);
+	const className = `.jiraSelectGitHubVersion__options__card.${selectedVersion}`;
+
+	if (selectedVersion === "cloud") {
+		$(".jiraSelectGitHubVersion__options__card.server").removeClass("selected");
+	}
+
+	if (selectedVersion === "server") {
```

Comment:
Sorry, "but the condition as displayed in the screen is, the user can not select both options." is a little confusing. Are you saying the user should be able to should be able to select both options?

Reply:
From the UI, it can be observed that the user can select only one option at a time. So, if we do multiple `if`, that means all those `if` will be checked. But if we do a `if-elseif`, then if the first condition is true, it won't do any other checks. It will go to the next `elseif` check only if the first `if` fails.

**Multiple if**
```
if (conditionA) -> do A
if (conditionB) -> do B
```
Here, if `conditionA` is true, then we don't need to check for `conditionB`. But since another `if` is being used, it will do that check, which will surely fail. It seems redundant.

So, instead we can do a `if-elseif`.
**if-elseif**
```
if (conditionA) -> do A
else if (conditionB) -> do B
```
So now if `conditionA` is true, then it won't do the other check. The only time if it would do the other check is if `conditionA` fails.

Reply:
lol I know how conditionals work. I meant the wording of your comment was confusing.

Reply:
ah apologies, I was just mentioning that both conditions can not be true, so its better to use `if-elseif`....